python/py-pew/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup			python 1.0
+PortGroup           github 1.0
+
+set package			pew
+set user			berdario
+version				1.1.0
+
+name				py-$package
+github.setup        $user $package $version
+github.tarball_from releases
+
+master_sites        ${github.homepage}/archive/
+distname            ${version}
+worksrcdir          ${github.project}-${version}
```

Comment:
These should be automatic with a github portgroup. If they are not, something is probably weird.

Reply:
I have never been able to get the GitHub port group to work; it looks for files in the wrong place. This one fails with
> Attempting to fetch pew-1.1.0.zip from  https://github.com/berdario/pew/releases/download/1.1.0

I found this hack in another person's Portfile.